pages/index.js

```diff
@@ -1,15 +1,38 @@
 import React, { Component } from 'react'
 import './root.scss'
 import CategoryList from './../components/CategoryList'
+import Rebase from 're-base'
+const base = Rebase.createClass('https://vivid-inferno-4672.firebaseio.com/')
+
+const getCategories = (categoryNames) => (
+  categoryNames.map(name => (
+    { icon: 'icon-home', name: name[0].toUpperCase() + name.slice(1) }
```

Comment:
Replace this with 

```javascript
String.prototype.capitalize = function() {
  this[0].toUpperCase() + this.slice(1);
}
```

then do `name.capitalize`

:trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: